packages/react/src/theme/components/iconButton.ts

```diff
@@ -1,54 +1,51 @@
-import { variantSolid, variantOutline, variantLink, variantGhost } from "./button";
-import { defineStyleConfig } from "@chakra-ui/react";
+import { defineStyle, defineStyleConfig } from "@chakra-ui/react";
 
-export const iconButtonTheme = defineStyleConfig({
-  baseStyle: () => ({
-    borderRadius: "6px",
-    outline: 0,
-    display: "flex",
-    alignItems: "center",
-    justifyContent: "center",
-    flexShrink: 0,
-    _focusVisible: {
-      boxShadow: "outline",
-    },
-    _disabled: {
-      opacity: 0.5,
-      cursor: "not-allowed",
-    },
-  }),
-  sizes: {
-    xs: {
-      padding: "6px",
-      gap: "6px",
-      width: "24px",
-      height: "24px",
-    },
-    sm: {
-      padding: "9px",
-      width: "32px",
-      height: "32px",
-    },
-    md: {
-      padding: "12px",
-      width: "40px",
-      height: "40px",
-    },
-    lg: {
-      lineHeight: "28px",
-      padding: "15px",
-      width: "48px",
-      height: "48px",
-    },
+const baseStyle = defineStyle({
+  borderRadius: "6px",
+  outline: 0,
+  display: "flex",
+  alignItems: "center",
+  justifyContent: "center",
+  flexShrink: 0,
+  _focusVisible: {
+    boxShadow: "outline",
   },
-  variants: {
-    primary: variantSolid,
-    secondary: variantOutline,
-    tertiary: variantLink,
-    ghost: variantGhost,
+  _disabled: {
+    opacity: 0.5,
+    cursor: "not-allowed",
   },
+});
+
+const sizes = {
+  xs: defineStyle({
+    padding: "6px",
+    gap: "6px",
+    width: "24px",
+    height: "24px",
+  }),
+  sm: defineStyle({
+    padding: "9px",
+    width: "32px",
+    height: "32px",
+  }),
+  md: defineStyle({
+    padding: "12px",
+    width: "40px",
+    height: "40px",
+  }),
+  lg: defineStyle({
+    lineHeight: "28px",
+    padding: "15px",
+    width: "48px",
+    height: "48px",
+  }),
+};
+
+export const iconButtonTheme = defineStyleConfig({
+  baseStyle,
+  sizes,
   defaultProps: {
-    variant: "primary",
+    variant: "solid",
     colorScheme: "green",
```

Comment:
Ser dette har ligget her fra før, men ikke sett default colorScheme her. Det er uansett satt til green for alle komponenter, så du trenger ikke sette det i det hele tatt så lenge det skal være grønn. Grunnen til at det ikke skal settes er for at det skla være lett å overskrive via prop i Kvib-providere.